main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-        4) window wiper controls
-    Name 2 things you can do with the:   Steering Wheel
-        1) adjust cruise control settings.
-        2) activate the car horn
-        
-Sub Object 2: Instrument Cluster
-    Name 4 things you'll find on the:   Instrument Cluster
-        1) mileage counter
-        2) fuel gauge 
-        3) speedometer
-        4) tachometer
-    Name 3 things you can do with the:   Instrument Cluster
-        1) check distance travelled
-        2) check fuel amount
-        3) measure speed of car
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1) temperature controls
-        2) AC activation button
-        3) fan level
-    Name 3 things you can do with the:   Environment Controls
-        1) adjust the temperature inside car
-        2) turn the AC off or on
-        3) adjust the amount of moving air produced by fan
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1) speaker volume control
-        2) on/off toggle switch for system
-        3) radio channel scan button
-    Name 3 things you can do with the:   Infotainment System
-        1) adjust speaker volume
-        2) switch infotainment system On
-        3) scan for radio channels
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) seat belt
-        2) seat horizontal position adjuster
-        3) seat back angle position adjuster
-    Name 2 things you can do with the:   Seat
-        1) adjust horizontal position of seat
-        2) adjust angle of seat back
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) Laptop
+5 properties:
+    1) number of letter keys
+    2) number of numeric keys
+    3) stores amount of binary information
```

Comment:
`stores` is a verb.  You have defined an action for your 3rd property.
- [ ] revise